style5.css

```diff
@@ -19,17 +19,17 @@
     height: auto;
     background-color: transparent;
     flex-direction: row;
-    /*padding-left: 50px;*/
     margin-bottom: 80px;
 }
 
 .row-1__text {
     display: flex;
     width: 100%;
     background-color: solid transparent;
-    border: 1px solid transparent;
+    border: 0px solid transparent;
```

Comment:
why we need border then? we can remove it

Reply:
already done